modules/cert-manager-enable-operand-log-level.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-log-levels.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-operand-log-level_{context}"]
+= Setting a log level for cert-manager
+
+You can set a log level for cert-manager to determine the amount of detail in logged error messages.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* The {cert-manager-operator} is installed.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager.operator cluster
+----
+
+. Set the log level value by editing the `spec.logLevel` section:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  creationTimestamp: "2023-05-10T07:34:15Z"
+  finalizers:
+  - cert-manager-operator.operator.openshift.io/cert-manager-controller-deployment
+  - cert-manager-operator.operator.openshift.io/cert-manager-webhook-deployment
+  - cert-manager-operator.operator.openshift.io/cert-manager-cainjector-deployment
+  generation: 1
+  name: cluster
+  resourceVersion: "47290"
+  uid: 0f5cd145-a20f-43d1-ac3d-0db28a6f9f06
```

Comment:
Fields creationTimestamp, resourceVersion, uid, et al ... are not needed to be shown.
```suggestion
apiVersion: operator.openshift.io/v1alpha1
kind: CertManager
...
```